Makefile

```diff
@@ -12,9 +13,24 @@ services:
 shell: services
 	$(APPDOCKER) /bin/bash
 
+shell-ui: services
+	$(UIDOCKER) /bin/bash
+
 test:
 	$(APPDOCKER) contrib/test.sh
 
+test-ui:
+	$(UIDOCKER) npm run test
+
+lint-ui:
+	$(UIDOCKER) npm run lint
+
+format-ui:
+	$(UIDOCKER) npm run format
+	
+format-check-ui:
+	$(UIDOCKER) npm run format:check
+
```

Comment:
I personally don't mind the lint targets.

The `PHONY` declaration is not doing much right now. But it's there to avoid conflicts with actual folder names. `build` directory is a common artefact in many processes. `services` used to be a top level directory in aleph containing `ingest-file` etc.

Reply:
> services used to be a top level directory in aleph containing ingest-file etc.

Ah, that makes sense!

Reply:
I have now moved these dependencies into a separate `requirements-dev.txt`! :)